x-pack/plugins/observability_solution/entity_manager/server/lib/entities/built_in/index.ts

```diff
@@ -7,7 +7,11 @@
 
 import { EntityDefinition } from '@kbn/entities-schema';
 import { builtInServicesEntityDefinition } from './services';
+import { builtInServicesPreaggEntityDefinition } from './services_preagg';
 
 export { BUILT_IN_ID_PREFIX } from './constants';
 
-export const builtInDefinitions: EntityDefinition[] = [builtInServicesEntityDefinition];
+export const builtInDefinitions: EntityDefinition[] = [
+    builtInServicesEntityDefinition,
```

Comment:
why we need both? raw transfactions and the aggregated?

Reply:
this is development setup for me to compare both outputs, we'll only merge one

Reply:
👍